LEGO1/lego/legoomni/src/build/legocarbuild.cpp

```diff
@@ -26,6 +27,12 @@
 #include <isle.h>
 #include <vec.h>
 
+// names and values verified by BETA10 0x1006d742
```

Comment:
we should add a comment here that we also have these constants in our generated actions files, for example:

https://github.com/isledecomp/isle/blob/master/LEGO1/lego/legoomni/include/actions/racecar_actions.h#L17

The original code probably didn't have that and if you can confirm they used macros like this instead we should use that, but adding a comment for reference would be good

Reply:
I cannot prove that they used macros; the assertions look like https://github.com/isledecomp/isle/blob/9ae3e78836b1336ee1777c82329cdb3cd5fcea71/LEGO1/lego/legoomni/src/build/legocarbuild.cpp#L947 and I am not sure how that would work with enums. I'll add the comment.

Reply:
I went for something a little more self-documenting, please tell me how you like it

Reply:
Looks good thanks!